docs/api-documentation/json/event-notifier-message.json

```
@@ -121,13 +121,38 @@
           "works_location_type": {
             "description": "Type of location where the work is taking place. Types: Footway, Carriageway, Verge, Cycleway, Footpath, Parking place, Bus stop or stand, Cycle hire docking station, Taxi rank",
             "type": "string"
+          },
+          "permit_conditions": {
+            "description": "Comma separated list of the Condition codes which are applicable to the work. Condition codes: NCT01a, NCT01b, NCT02a, NCT02b, NCT04a, NCT04b, NCT05a, NCT06a, NCT07a, NCT08a, NCT08b, NCT09a, NCT09b, NCT09c, NCT10a, NCT11a, NCT11b, NCT12a",
+            "type": "string"
+          },
+          "road_category": {
+            "description": "The category of the road where the work is taking place.",
+            "type": "string"
+          },
+          "is_traffic_sensitive": {
+            "description": "Is work happening during traffic sensitive times.",
+            "type": "string"
+          },
+          "is_deemed": {
+            "description": "Did the work deem",
```

Thinking we should give a bit more context for this as users of open data might not be aware of what deem means in the context of work? 
```suggestion
            "description": "Did the work automatically deem ? (deadline date of assessment was passed)",
```